examples/reference/widgets/Toggle.ipynb

```diff
@@ -16,37 +16,36 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The `Toggle` widget allows toggling a single condition between ``True``/``False`` states. This widget is interchangeable with the `Checkbox` widget. \n",
+    "The `Toggle` widget allows users to toggle between `True` and `False` states using a button-style interface. This widget provides the same functionality as the `Checkbox` and `Switch` widgets but with a more prominent visual appearance that's perfect for actions and feature toggles.\n",
```

Comment:
`toggle.disabled` not clear visually
`toggle.loading` not working

![image](https://github.com/user-attachments/assets/02ba8137-896e-4a50-8b23-70755f8ededa)


```python
import panel_material_ui as pmui
pmui.Toggle(label="contained", variant="contained", value=True, disabled=True, loading=True).servable()
```

Reply:
Unclear to me what to do about `disabled`, Mui is control.

Reply:
`loading` is fixed.